template/tsconfig/vitest/tsconfig.vitest.json

```diff
@@ -1,11 +1,18 @@
 {
   "extends": "./tsconfig.app.json",
+  "include": [
+    "src/**/__tests__/*"
+  ],
   "exclude": [],
   "compilerOptions": {
     "composite": true,
+    "outDir": "./node_modules/.tmp/vitest",
     "tsBuildInfoFile": "./node_modules/.tmp/tsconfig.vitest.tsbuildinfo",
```

Comment:
tsBuildInfoFile can be removed since now that we set outDir, its default value will be <outDir>/<config name>.tsbuildInfo, compare https://www.typescriptlang.org/tsconfig/#tsBuildInfoFile